Profiler/ProfilerPlugin.cs

```diff
@@ -13,8 +14,16 @@ public class ProfilerPlugin : TorchPluginBase
         /// <inheritdoc cref="TorchPluginBase.Init"/>
         public override void Init(ITorchBase torch)
         {
+            base.Init(torch);
+            this.ListenOnGameLoaded(() => OnGameLoaded());
+
             var pgmr = new ProfilerManager(torch);
             torch.Managers.AddManager(pgmr);
         }
+
+        void OnGameLoaded()
```

Comment:
What is this code doing?

Reply:
Not sure. Probably I tried to preserve what existed in the code base that I didn't understand at the time and left it there. Seems like it's dead so I'll remove it

Reply:
(Didn't see anything change when I ran the game without that code)